mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -1,9 +1,17 @@
 CLASS net/minecraft/class_77 net/minecraft/client/particle/Particle
+	FIELD field_2635 textureId I
+	FIELD field_2636 prevU F
+	FIELD field_2637 prevV F
+	FIELD field_2638 particleAge I
 	FIELD field_2639 maxAge I
```

Comment:
Would maybe be good to name this `maxParticleAge` to be consistent with `particleAge`

Reply:
agree